digitalforms-api/src/main/java/ca/bc/gov/open/pssg/rsbc/digitalforms/controller/ScheduleReviewController.java

```diff
@@ -0,0 +1,71 @@
+package ca.bc.gov.open.pssg.rsbc.digitalforms.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import ca.bc.gov.open.pssg.rsbc.digitalforms.model.JSONResponse;
+import ca.bc.gov.open.pssg.rsbc.digitalforms.model.ReviewTimeAvailabilityInfo;
+import ca.bc.gov.open.pssg.rsbc.digitalforms.model.ReviewTimeSlot;
+import ca.bc.gov.open.pssg.rsbc.digitalforms.service.ScheduleReviewService;
+import ca.bc.gov.open.pssg.rsbc.digitalforms.util.DigitalFormsConstants;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+/**
+ * 
+ * Schedule a review Controller.
+ * 
+ * @author sivakaruna
+ *
+ */
+@RestController
+@Api(value = "Schedule Review", tags = { "Schedule Review" })
```

Comment:
Can we change this to "Review Scheduling"? I think it better reflects the two operations which will be included in this tag. The other operation will be the saving of the chosen review schedule time.

Reply:
updated

Reply:
Thank you.